### PR TITLE
On import, compute the eliminating team.

### PR DESCRIPTION
**Pull request by EboMike**
If one team ends up eliminating the other, store that in the database so it gets a 10k bonus.

### CODE REVIEW

File: helpers/sm5helper.py [hunk 1]
```diff
@@ -590,6 +590,52 @@ async def get_sm5_rating_over_time(entity_id: str, min_time: datetime = _MIN_DAT
...
+        return game
+
+    eliminating_team = await get_sm5_last_team_standing(game)
+
+    if not eliminating_team:
+        # No elimination. Do nothing.
+        return game
+
+    # Add the bonus.
+    game.scores
```
Comment (spookybear0) [inline helpers/sm5helper.py:634]: This part is incomplete, it never adds the scores. Additionally, we should add the 10k bonus dynamically when the total is being displayed for clarity ex: 37203 (+10000).

Comment (spookybear0) [reply helpers/sm5helper.py:634]: It also seems to be unused here and in #101

Comment (EboMike) [reply helpers/sm5helper.py:634]: lol, I forgot to remove that. Yes, that was the initial approach, but then I changed it after we chatted about how to best implement it. It is already being done dynamically in get_team_score() in sm5.py, you can see it there. So we store the score without the adjustment but we add it whenever the code actually fetches it.

Comment (EboMike) [reply helpers/sm5helper.py:634]: Btw, if you want to show the adjustment separately, we can easily do that. We can add a separate get_score_adjustment() function in sm5.py and get_team_score() will call that. We call that function also when displaying a score card and if it's non-zero, we show it next to score. I can do that in a separate CL, unless you want to have it right now first.

**Pull request merged**